torchchat/usages/openai_api.py

```diff
@@ -288,13 +289,50 @@ def __init__(self, *args, **kwargs):
                 else self.model.text_transformer_args.max_seq_length
```

Comment:
One of the things on our list is having a unification configuration system for both tune-backend model and chat-backend models to get rid of the try .. except here.